tests/Mvp.Selections.Api.Tests/Services/LicenseServiceTests.cs

```diff
@@ -0,0 +1,223 @@
+﻿using System.IO.Compression;
+using System.Net;
+using AutoFixture;
+using FluentAssertions;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Mvp.Selections.Api.Model;
+using Mvp.Selections.Api.Services;
+using Mvp.Selections.Data.Repositories;
+using Mvp.Selections.Data.Repositories.Interfaces;
+using Mvp.Selections.Domain;
+using NSubstitute;
+using Xunit;
+
+namespace Mvp.Selections.Api.Tests.Services
+{
+    public class LicenseServiceTests
```

Comment:
We are missing UnitTests for the `DownloadLicenseAsync` functionality, please add tests for all happy path and all failure paths in that method